.gitignore

```diff
@@ -5,6 +5,7 @@ spec/dummy/db/*.sqlite3
 spec/dummy/db/*.sqlite3-journal
 spec/dummy/log/*.log
 spec/dummy/tmp/
+.idea
```

Comment:
Is this necessary?

Reply:
i think so, it's the config folder for IDEA products, otherwise it may be commited by mistake